app/src/main/res/layout/contract_detail_documents_document.xml

```diff
@@ -1,10 +1,58 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Button xmlns:android="http://schemas.android.com/apk/res/android"
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    style="?materialButtonContainedLargeStyle"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginStart="@dimen/base_margin_double"
-    android:layout_marginTop="@dimen/base_margin_double"
-    android:layout_marginEnd="@dimen/base_margin_double"
-    tools:text="@string/MY_DOCUMENTS_INSURANCE_CERTIFICATE" />
+    android:layout_marginBottom="@dimen/base_margin"
+    android:layout_height="@dimen/contract_document_card">
```

Comment:
I still hold that we should allow this height to come naturally from its contents, not hard-coded 😉

Reply:
What is the reason behind that?

Reply:
Because if circumstances cause the contents to become higher than the container, then the overflow will be cut off.